settings.py

```diff
@@ -3,7 +3,25 @@
 
 logger = logging.getLogger(__name__)
 
-LOGGING_FORMAT = "%(asctime)s|%(levelname)s: sdx-sequence: %(message)s"
+LOGGING_FORMAT = "%(message)s"
```

Comment:
It's the preferable format for Splunk as it'll be JSON. There might be an issue with getting Splunk to recognize JSON in one case and the old format for other services simultaneously, I'm checking on that this morning. May have to revert the change sorry! :/